README.md

```diff
@@ -1,35 +1,34 @@
 # [`otmonitor`](http://otgw.tclcode.com/otmonitor.html) Home Assistant supervisor add-on
 
-Install by going to Supervisor -> Add-on store -> Add new repository by url and fill in `https://github.com/basnijholt/addon-otmonitor`.
```

Comment:
I prefer to keep this line in. Instead of the lines that you removed below, let's link to https://github.com/basnijholt/addon-otmonitor/blob/master/otmonitor/DOCS.md.